LoopKitUI/Extensions/Environment+InvestigationalDevice.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Environment+InvestigationalDevice.swift
+//  LoopKitUI
+//
+//  Created by Cameron Ingham on 7/2/24.
+//  Copyright © 2024 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+private struct InvestigationalDeviceKey: EnvironmentKey {
```

Comment:
Any reason this needs to live in LoopKit?  If it is only used in Loop, then it probably belongs in Loop.

Reply:
Just mirroring the appName environment value. I can certainly move it if you'd like!

Reply:
This would allow TidepoolOnboarding to use this in the case that it's decided we need a warning added there as well.